app/styles/ui/dialogs/_merge-conflicts.scss

```
@@ -71,6 +71,12 @@ dialog#rebase-conflicts-list {
         display: flex;
         align-items: center;
         justify-content: center;
+
+        .resolve-arrow-menu {
+          .octicon {
+            margin-left: var(--spacing-half);
```

While you're in here could I possibly convince you to apply the same size constraints for the arrow that we have for the other button?

Either by copying and pasting:
```suggestion
            margin-left: var(--spacing-half);
            width: 10px;
            height: 14px;
```

Or by changing the class name from `resolve-arrow-menu` to just `arrow-menu` and letting the other styles apply here as well.

---

I ended up taking in this suggested change because I wasn't sure about the best way to keep the `margin-left` style around and leverage `arrow-menu` (need to differentiate between a manual conflict and a file with conflict markers, but there are no suitable selectors currently).